include/glow/ExecutionEngine/ExecutionEngine.h

```diff
@@ -159,6 +159,17 @@ void runBatch(ExecutionEngine &EE, PlaceholderBindings &bindings,
               llvm::ArrayRef<Placeholder *> ph, llvm::ArrayRef<Tensor *> inputs,
               llvm::StringRef name = "");
 
+/// Runs \p numMinibatchRuns iterations of the compiled function called \p name.
+/// The method updates a global counter and future invocations of this method
+/// continue running iterations of the batch at the next available slice.
+/// Each sample output is copmared VS ground-truth \p labelsInput, and
+/// \p numCorrectEstimates counter is updated correspondingly.
+void evalBatch(ExecutionEngine &EE, PlaceholderBindings &bindings,
```

Comment:
Why do you put it into ExecutionEngine.h, which is a very generic header, unrelated to testing? Wouldn't some kind of TestUtils.h be a better place for it?

Reply:
This helper is very similar to runBatch (and is useful after using runBatch to train a model) so I thought it makes sense to place them together.